hazelcast/src/main/java/com/hazelcast/spi/tenantcontrol/TenantControl.java

```diff
@@ -33,8 +31,7 @@
  * @author lprimak
  */
 @Beta
-public interface TenantControl extends Serializable {
-
+public interface TenantControl extends DataSerializable {
```

Comment:
This will break any implementation of TenantControl. In theory it is allowed because it is @Beta, but something that needs to be considered since there is already a release with this interface.

Reply:
TenantControl was created by me for Payara. I doubt very much it'll break anyone else. Besides, many, many things were changed in more of an incompatible way from Hazelcast 3.x to 4.x

Reply:
Hazelcast 4.0 is a major release and hence allowed to break API.

Reply:
Generally since it's beta and since it's a rather obscure feature, I'm ok with changing it in a minor release.